db/db_impl.cc

```diff
@@ -645,6 +668,7 @@ Status DBImpl::SyncWAL() {
     need_log_dir_sync = !log_dir_synced_;
   }
 
+  TEST_SYNC_POINT("DBWALTest::SyncWALNotWaitWrite:1");
```

Comment:
Yeah I could not find anything either. I updated the sync documentation of logs_ to clarify that.